students/jimin/users/views.py

```diff
@@ -1,3 +1,45 @@
-from django.shortcuts import render
+import json ,re
 
+from django.http     import JsonResponse
+from django.views    import View
+
+
+from users.models    import User
 # Create your views here.
+
+class SignUpView(View):
+    def post(self, request):
+        try :
+             data          = json.loads(request.body)
+             email         = data['email']
+             password      = data['password']
+             phone_number  = data['phone_number']
+             
+             
+            
+             regex_email        = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9_-]+\.[a-zA-Z0-9-.]+$' #(소문자 a~z, 대문자 A~Z, 숫자 0~9, +-_.) + @ + .
+             regex_password     = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,25}$'
+             regex_phone_number = '\d{3}-\d{4}-\d{4}'
```

Comment:
해당 정규식은 변수가 아닌 상수에 가깝습니다. 파이썬에서 상수의 의미로 선언을 할 때 대문자로 구성합니다.

```suggestion
             REGEX_EMAIL        = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9_-]+\.[a-zA-Z0-9-.]+$' #(소문자 a~z, 대문자 A~Z, 숫자 0~9, +-_.) + @ + .
             REGEX_PASSWORD     = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,25}$'
             REGEX+PHONE_NUMBER = '\d{3}-\d{4}-\d{4}'
```